anyway/widgets/widget_utils.py

```diff
@@ -128,3 +128,29 @@ def format_2_level_items(
             series_data.append({LKEY: l2, VAL: num})
         res.append({LKEY: l1, SERIES: series_data})
     return res
+
+
+def order_by_list_stack_bar_widget(
+    structured_data_list: List[Dict[str, Union[str, List]]], severity_order: List[str]
+) -> List[Dict[str, Union[str, List]]]:
+    severity_dict = {}
+    for struct in structured_data_list:
+        series_data = struct["series"]
+        for injury in series_data:

```

Comment:
according to the types defined (Union[str,List]), this might be a either a string or a list and so i don't understand this enumeration - if it's a string you iterate over chars and injury.items() will fail, same for iterating over a list